app/views/placements/schools/placements/build/check_your_answers.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= content_for :page_title, sanitize(t(".title")) %>
+<%= render "placements/schools/primary_navigation", school: @school, current_navigation: :placements %>
+
```

Comment:
You're missing a back link.
Prototype:
https://school-placements-beta-dev-73591d8bbe5f.herokuapp.com/organisations/049fc44e-6876-4884-8a77-d363173ce8d2/placements/new/check
![screencapture-school-placements-beta-dev-73591d8bbe5f-herokuapp-organisations-049fc44e-6876-4884-8a77-d363173ce8d2-placements-new-check-2024-04-17-13_28_04](https://github.com/DFE-Digital/itt-mentor-services/assets/17162488/d709b9a5-8a9f-429b-8d88-bcaf6ec17e3d)